public/index.html

```diff
@@ -0,0 +1,423 @@
+<!doctype html>
+<html class="no-js" lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Avengers</title>
+    <meta name="viewport" content="width=device-width, user-scalable=no">
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <!-- menu -->
+    <nav class="nav top-nav">
+      <ul class="flex-container flex-navbar">
+        <li class="flex-nav-item"><a href="#home">Home.</a></li>
+        <li class="flex-nav-item"><a href="#services">Services.</a></li>
+        <li class="flex-nav-item"><a href="#portfolio">Portfolio.</a></li>
+        <li class="flex-nav-item"><a href="#about">About.</a></li>
+        <li class="flex-nav-item"><a href="#team">Team.</a></li>
+        <li class="flex-nav-item"><a href="#contact">Contact.</a></li>
+      </ul>
+    </nav>
+    <!-- header -->
+    <section class="header flex-container header-notch">
+      <h1>
+        <span class="icon-droplet"></span><br>
+        Hi there! We're Avengers, <strong>creative design agency</strong>.
+        We build <strong class="underline">awesome</strong> mobile and websites.
+      </h1>
+    </section>
+    <!-- services  -->
+    <article class="services" id="services">
+      <div class="horizontal-rules">
+        <span class="rule-L-long">&nbsp;</span>
```

Comment:
What happened to the `:before` and `:after` pseudo-content on this? Separate elements is a step backward from what you had previously. Was there a reason for the change?